### ISSUE TITLE
Permission to perform this operation was denied

### ISSUE THREAD

**Issue by Jasin2069** [vmware/vic#7734]
I am trying to deploy VIC in our lab for testing. I have the OVA for 1.3.1 deployed on vsphere 6.5. I have created a user VIC as administrator. In the HTML5 GUI deploy a VCH with default settings. I disable client auth for now while I'm just trying to get it working. I assign it a static IP. 

`vic-machine-windows create --name VHC1 --compute-resource /SiN/host/SiN --cpu-reservation 1 --cpu-shares normal --memory-reservation 1 --memory-shares normal --endpoint-cpu 2 --image-store Local7/Containers --base-image-size 8GB --volume-store Local7/ContainerDefault:default --bridge-network ContainerBridge1 --bridge-network-range 172.16.0.0/12 --public-network Management --public-network-ip .38/24 --public-network-gateway 10.1 --dns-server 10.--dns-server 10. --tls-cname VHC1 --organization SiN --certificate-key-size 2048 --no-tlsverify --user J --thumbprint 24:4 --target vcenter./SiN --ops-user VIC@ --ops-grant-perms`

In admiral I add the VHC and try and provision anything - postgres for example. 

I get in the eventlog 

`Service https://.38:2376/v1.24/containers/create?name=postgres-mcm1008-72041191351 returned error 500 for POST. id 13447716 message Server error from portlayer: ServerFaultCode: Permission to perform this operation was denied.; Reason: {"message":"Server error from portlayer: ServerFaultCode: Permission to perform this operation was denied."}`

I tried the same thing with docket cli. I've redeployed VHC multiple times.

> docker -H 38:2376 --tls  run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -d postgres

```
docker.exe: Error response from daemon: Server error from portlayer: ServerFaultCode: Permission to perform this operation was denied.
See 'C:\Program Files\Docker\docker.exe run --help'.
```

**VIC version:**
1.3.1
**Steps to reproduce:**
Try to provision anything (HelloWorld runs)
**Actual behavior:**
Permission to perform this operation was denied

**Logs:**
[container-logspostgres.zip](https://github.com/vmware/vic/files/1908710/container-logspostgres.zip)

**Comment by hickeng** [comment]
After some direct conversation, this was concluded by addition of the `VirtualMachine->Configuration->Add Existing Disk` privilege to the `vic-vch-datacenter` role.

**Comment by hickeng** [comment]
@anchal-agrawal This was with 1.3.1. I know you've been testing with 1.4.0-dev and ops-user. Can you close this issue if it's resolved in the 1.4.0-dev `ops-user-grant` logic.

**Comment by anchal-agrawal** [comment]
@Jasin2069 Thanks for reporting this issue!

@Jasin2069 @hickeng I've verified that this is fixed in our upcoming release. The `VirtualMachine->Configuration->Add Existing Disk` privilege (present in the `vic-vch-endpoint` role) is now applied at the resource pool _and_ the VCH folder (more granular than when applied at the datacenter-level). I tested by removing the `VirtualMachine->Configuration->Add Existing Disk` privilege from the [Endpoint role](https://github.com/vmware/vic/blob/master/lib/install/opsuser/opsuser_conf.go#L76-L100) - doing so caused the `permission denied` error when I attempted to create a `postgres` container, which I noticed creates an anonymous volume (hence the need for the `VirtualMachine->Configuration->Add Existing Disk` privilege).
